tests/query/atomWithQuery.test.tsx

```diff
@@ -5,307 +5,309 @@ import { atomWithQuery } from '../../src/query'
 import { getTestProvider } from '../testUtils'
 import fakeFetch from './fakeFetch'
 
-const Provider = getTestProvider()
+describe(atomWithQuery.name, function () {
```

Comment:
wrapping in `describe` is generally best practice

Reply:
I don't disagree about using `describe`. (Not sure about `.name` pattern).

Would you create a separate PR for this change? I mean if we prefer `describe` generally, should we change all tests?

Reply:
> Would you create a separate PR for this change? I mean if we prefer describe generally, should we change all tests?
Sure! makes sense to be consistent 

> (Not sure about .name pattern)
I've found this is nice if ever a refactor happens and the method changes.

Reply:
Sounds good.